src/components/FeatureViews/Accounts/TabComponents/InstructorSchedule.js

```diff
@@ -67,23 +90,23 @@ const Schedule = ({instructorID}) => {
                 {hoursWorked} hour(s) worked this month
             </h3>
             <FullCalendar
```

Comment:
add `minTime` attribute to be "07:00:00" like the FullCalendar component in scheduler. This will set the earliest viewable time to be 7am.

Reply:
[minTime Documentation](https://fullcalendar.io/docs/minTime)